packages/InputGroup/InputGroup.jsx

```diff
@@ -26,7 +26,6 @@ const InputGroup = ({
 }) => {
   const inputRef = createRef()
   const [inputValue, setInputValue] = useState('')
-  const inputId = useMemo(() => (label ? uniqueId(label) : id), [label, id])
```

Comment:
Rather than make `id` required, could this instead prefer to use the user-supplied id?

```diff
- const inputId = useMemo(() => (label ? uniqueId(label) : id), [label, id])
+ const inputId = useMemo(() => id || label && uniqueId(label) || undefined, [label, id])
```

Also, what should happen if the user doesn't supply a `label`, how will the customer know this input's label?

Reply:
@theetrain 

1. Good to know. I will make the change
2. Make sense. I will let designer know about your feedback regarding the opitonal label. In the meantime, I will update the